bin/gremlin.sh

```diff
@@ -1,22 +0,0 @@
-#!/bin/bash
```

Comment:
It's no longer a file, it's a symbolic link. GitHub only makes it look like a normal file.